### PR TITLE
UI refactor

### PR DESCRIPTION
**Pull request by cmsigrist**
Refactor all the buttons, simplifies some of the layouts (and also change the instanciation of the viewModel, as certain activity did not use a newly added function)

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -165,6 +165,7 @@
     <string name="title_activity_weather">WeatherActivity</string>
     <string name="edit_gender">Edit</string>
     <string name="update_gender">Update Gender</string>
+    <string name="policy">By clicking on Log In, you agree with our Terms. Learn how we process you data in our Privacy Policy and Cookies Policy.</string>
     <string name="menu_refresh">Refresh</string>
     <string name="weather_update_failed">Can\'t get weather data now, try again later</string>
     <string name="fahrenheit_temperature">%d °F</string>
```
Comment (Lollipoke) [inline app/src/main/res/values/strings.xml:171]: Could be interesting to remove all unused string if you can

**Review by Lollipoke — APPROVED** [review]
A few details, nothing major :) Great job !

File: app/src/main/res/layout/activity_recording.xml [hunk 2]
```diff
@@ -1,56 +1,52 @@
...
 
-        <Chronometer
-            android:id="@+id/recordTimer"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_centerVertical="true"
-            android:layout_marginStart="60dp"
-            android:layout_toEndOf="@id/recordingButton" />
-    </RelativeLayout>
+    <ImageButton
```
Comment (cgntnr) [inline app/src/main/res/layout/activity_recording.xml:33]: Could be nice to add contentDescription for this as Cirrus suggests.

File: app/src/main/res/layout/activity_settings.xml [hunk 3]
```diff
@@ -198,6 +200,7 @@
         android:layout_height="50dp"
         android:layout_below="@id/divider_distance"
         android:background="@color/transparent"
+        android:foreground="?selectableItemBackground"
         android:onClick="startShowMeSettings" />
 
     <TextView
```
Comment (cgntnr) [inline app/src/main/res/layout/activity_settings.xml:206]: For line 227 as well, which was missing from before.

File: app/src/main/res/layout/fragment_audio_player.xml [hunk 4]
```diff
@@ -37,17 +37,6 @@
             android:layout_alignParentStart="true"
             tools:layout_marginLeft="20dp" />
```
Comment (cgntnr) [inline app/src/main/res/layout/fragment_audio_player.xml:39]: Again at Line 21, would be nice to add contentDescription.

File: app/src/main/res/values/colors.xml [hunk 5]
```diff
@@ -20,4 +20,5 @@
     <color name="blindly_purple">#A786E3</color>
     <color name="blindly_purple_secondary">#9166DD</color>
     <color name="blindly_blue">#93C6F6</color>
+    <color name="blindly_grey">#8D8C8F</color>
 </resources>
```
[the remark this answers is not in the document]
Comment (cgntnr) [reply app/src/main/res/values/colors.xml:24]: We actually tried to remove colors before and even though we didn't use, we had problems when we removed them if I remember correctly.

**Review by cgntnr — APPROVED** [review]
Added couple of comments, not much to say on top of what @Lollipoke said. Good refactoring !

File: app/src/main/res/values/colors.xml (hunk 5 above)
[the remark this answers is not in the document]
Comment (Lollipoke) [reply app/src/main/res/values/colors.xml:24]: Oh okay ! Better leave them then :p

File: app/src/main/res/layout/activity_edit_gender.xml [hunk 6]
```diff
@@ -54,15 +54,15 @@
...
-        android:layout_height="30dp"
+        android:layout_width="@dimen/small_pill_shape_width"
+        android:layout_height="@dimen/small_pill_shape_height"
         android:layout_alignBottom="@id/gender_radio_group"
         android:layout_toEndOf="@id/gender_radio_group"
         android:layout_marginStart="50dp"
-        android:layout_marginBottom="10dp"
+        app:shapeAppearanceOverlay="@style/PillShapeButton"
         android:text="@string/edit_gender"
         android:visibility="invisible"/>
```
[the remark this answers is not in the document]
Comment (cmsigrist) [reply app/src/main/res/layout/activity_edit_gender.xml:68]: Weirdly enough when I give an input type it "overrides" the textAllCaps, I'm afraid I'll have to ignore this warning :(

File: app/src/main/res/values/colors.xml (hunk 5 above)
[the remark this answers is not in the document]
Comment (cmsigrist) [reply app/src/main/res/values/colors.xml:24]: I think they're used somewhere in a default theme :/

**Pull request merged**